caffe2/python/pybind_state.cc

```diff
@@ -1237,13 +1239,15 @@ void addGlobalMethods(py::module& m) {
          const std::map<std::string, std::vector<TIndex>> blob_dimensions) {
         // Parse protobuffers to NetDefs
         std::vector<std::unique_ptr<caffe2::NetDef>> nets;
```

Comment:
Same here.

Reply:
same reason. If you see another way of fixing the issue, please let me know!